src/api/app/assets/stylesheets/webui2/user.scss

```diff
@@ -1,3 +1,39 @@
 #home-login {
   color: $gray-500;
 }
+
+.activity {
+}
+
+.activity-percentil0 {
+  background-color: #dcd5da;
+}
+
+.activity-percentil1 {
+  background-color: #f9b4dd;
+}
+
+.activity-percentil2 {
+  background-color: #ff5cb3;
+}
+
+
+.activity-percentil3 {
+  background-color: #ec1596;
+}
+
+
+.activity-percentil4 {
+  background-color: #c60b7b;
+}
+
+#contributors-table {
+  td {
+    padding: 0;
+    border: 1px solid white;
+  }
+  td.week-day {
+    width: 1px;
```

Comment:
I would stick with rem. For once to not mix different font size types and also because px doesn't scale depending on the screen size

Reply:
not scaling is exactly the point. We want this to have minimal width